One way to include figures

iamaleksandra opened the pull request:
These set of files is primarily for fun and considerations. Let me know if I'm doing something improperly regarding this service.

iamaleksandra committed:
One way to include figures

These set of files is primarily for fun and considerations. Let me know if I'm doing something improperly regarding this service.

hombit commented on misc/book/converter_with_figs.py:
```diff
@@ -0,0 +1,63 @@
+    with open(file_name, 'r', encoding='utf-8') as my_file:
+        my_code = my_file.readlines()
+
+    if 'import matplotlib.pyplot as plt\n' in my_code:
```
This is not a very safe method of Python code running. Please, see `runpy`, `importlib`, `code` and `parser` standard library modules.

iamaleksandra committed:
Update converter_with_figs.py

iamaleksandra committed:
Update converter_with_figs.py

Renaming figures that belong to one script in order to do not mind about it while writing script itself

iamaleksandra committed:
Update converter_with_figs.py
